tests/lib/fdtable/src/main.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <zephyr.h>
+#include <sys/fdtable.h>
+
+void test_z_reserve_fd(void)
+{
+
+    int fd = -1;
```

Comment:
No need to assign an initial value here, since it will always be overwritten by what z_reserve_fd() returns